README.md

```diff
@@ -13,46 +13,47 @@ Znaczy [pull requestów](https://github.com/nurkiewicz/polski-w-it/pulls), nie t
 
 | angielski            | polskawy            | polski                                |
 | :------------------- | :------------------ | :------------------------------------ |
-| approve              | zapruwować          | zaakceptować                          |
+| approve              | zaapruwować         | zaakceptować                          |
 | authentication       | autentykacja        | uwierzytelnienie (!)                  |
 | authorization        |                     | autoryzacja                           |
 | build                | bild                | budowanie                             |
-| button               | baton               | przycisk                              |
-| cancel               | kanselować          | anulować                              |
+| button               | baton               | przycisk, guzik                       |
+| cancel               | kanselować, anulacja| anulować                              |
 | case                 | kejs                | przypadek                             |
 | confirm              | konfirmować         | potwierdzić                           |
-| content              |                     | zawartość, treść                      |
+| content              | kontent             | zawartość, treść                      |
```

Comment:
W myśl https://github.com/nurkiewicz/polski-w-it/pull/23 chciałbym unikać wersji "polskawej", która jest tylko zapisem fonetycznym. Natomiast samo pojęcie _content_ jak najbardziej spoko 👍